Fix Translation

Riffaells opened the pull request:
Updated the Russian localization and partially updated the English and Chinese localizations.
Fixed hardcoded localization strings and updated date formatting.


Translation for convenience:
更新了俄语本地化，并部分更新了英语和中文本地化。
修复了本地化中的硬编码字符串，并更新了日期格式。

Riffaells committed:
update ru lang

Riffaells committed:
fix translation

Riffaells committed:
Removing the localization hardcode

Newdea commented:
hi, thanks for your pr, but #119  WIP ， can you update this pr after #119 merged into dev-mater branch

Riffaells commented:
#121 Ready @Newdea

Newdea committed:
cherry pick card parse (#119)

- closes #98
- closes #118

Commit: 06a93160284c5f4cb6002410b6ef63ee4030defe
Parents: a4b88dc01d04bd8893605d0abacfbb5edc728d82
Author: Stephen Mwangi <mail@stephenmwangi.com>
Committer: GitHub <noreply@github.com>
Date: Thu Sep 26 2024 13:42:30 GMT+0800 (China Standard Time)

chore: add linting for camelcase variables (#1098)

Commit: 657e37d316c2cf2d31e07af9f3e071e25b2d2c8f
Parents: b4f976598d7fc14a9c3789ec8a37f182d68b20a8
Author: Stephen Mwangi <mail@stephenmwangi.com>
Committer: GitHub <noreply@github.com>
Date: Sun Sep 22 2024 18:14:40 GMT+0800 (China Standard Time)

fix: parsing code blocks & custom separators (#1081)

---------

Co-authored-by: Newdea <Newdea@users.noreply.github.com>

Newdea commented:
基于这个分支做了点修改,就合并这个了

Newdea committed:
Merge branch 'dev-master' into dev-master

Newdea committed:
lint

Newdea committed:
lint
